### ISSUE TITLE
server database refresh?

### ISSUE THREAD

**Issue by strowi** [aquasecurity/trivy#448]
Hi,

the readme states:
```
--only-update, --refresh and --auto-refresh are deprecated since they are unnecessary now. These options will be removed at the next version
```

Does this mean running "trivy server" will regularly update the database by itself?

thx + regards,
strowi

**Comment by simar7** [comment]
Hi @strowi, yes that's correct you can see the code here https://github.com/aquasecurity/trivy/blob/18b80e3781a8c8d1f4365071a3d0866e3ec5e81d/pkg/rpc/server/listen.go#L50-L59

**Comment by strowi** [comment]
thx @simar7 , i guess i somehow stumbled into #452 which has been fixed in 0.5.4;)
